page-data.php

```diff
@@ -23,6 +23,7 @@
     ?>
 
 <div id="js-TJIChartView" class="row">
+	<h2>Current Data Set: <span class="tji-current-dataset"></span></h2>
```

Comment:
I think we can remove this line. Currently, TJIChartView obliterates everything inside the `js-TJIChartView` div once the data loads. TJIChartView also already generates an element displaying the number of records.